src/screens/FeedScreen/ContentPoll.js

```diff
@@ -243,14 +244,14 @@ const stylesComponent = (pollLength) =>
       marginBottom: 0
     },
     pollOptionItemContainer: {
-      backgroundColor: COLORS.lightgrey,
+      backgroundColor: colors.lightgrey,
       marginBottom: 8,
       borderRadius: 8,
       display: 'flex',
       flexDirection: 'row'
     },
     pollOptionItemContainerActive: {
-      backgroundColor: COLORS.holytosca30percent,
+      backgroundColor: colors.holytosca30percent,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated from `COLORS.holytosca30percent` to `colors.holytosca30percent`. Make sure that `colors.holytosca30percent` is defined and is the intended color.
```diff
- backgroundColor: COLORS.holytosca30percent
+ backgroundColor: colors.holytosca30percent
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->